cmd/sqlcmd/main.go

```diff
@@ -58,6 +58,7 @@ type SQLCmdArguments struct {
 	ColumnSeparator             string            `short:"s" help:"Specifies the column separator character. Sets the SQLCMDCOLSEP variable."`
 	ScreenWidth                 *int              `short:"w" help:"Specifies the screen width for output. Sets the SQLCMDCOLWIDTH variable."`
 	TrimSpaces                  bool              `short:"W" help:"Remove trailing spaces from a column."`
+	MultiSubnetFailover         bool              `short:"M" help:"Provided for backward compatibility."`
```

Comment:
Is it possible to add a long: parameter to give a bit more detail on what it means to be "provided for backward compat".... in the long we say this option is a no-op, does nothing, and it provided to not break existing scripts.  The short: right now doesn't tell the user that this does nothing.